config_parse.go

```
@@ -28,6 +30,11 @@ func ReadProxyConfig(path string) (c ProxyConfig, err error) {
 		return
 	}
 
+	err = envconfig.Process("veneur", &c)
+	if err != nil {
+		log.Fatal(err.Error())
```

We should just return the error here, right? Why would we `log.Fatal`?